docs/integration/java/ShardingSphere.md

```diff
@@ -0,0 +1,46 @@
+---
+title: ShardingSphere
+description: Using Casdoor in ShardingSphere
+keywords: [ShardingSphere]
+author: jakiuncle
+---
+
+[shardingsphere-elasticjob-ui](https://github.com/apache/shardingsphere-elasticjob-ui) have integrated Casdoor.We can use it after config it.
+
+## Step1. Deploy Casdoor
+
+Firstly, the Casdoor should be deployed.
+
+You can refer to the Casdoor official documentation for the [Server Installation](/docs/basic/server-installation).
+
+After a successful deployment, you need to ensure:
+
+- The Casdoor server is successfully running on **http://localhost:8000**.
+- Open your favorite browser and visit **http://localhost:7001**, you will see the login page of Casdoor.
+- Input `admin` and `123` to test login functionality is working fine.
+
+Then you can quickly implement a casdoor based login page in your own app with the following steps.
+
+## Step2. Configure Casdoor application and configure application in ShardingSphere
+
+1.**Create or use an existing Casdoor application**
+![img](../../../static/img/integration/java/ShardingSphere/casdoorConfig.png)
```

Comment:
it's ok, the advantage of this is that you can preview the effect in the markdown editor, right? But usually we write `/img/integration/java/ShardingSphere/casdoorConfig.png`. I recommend you to use the usual way. What do you think @leo220yuyaodog @hsluoyz

Reply:
@Selflocking agreed, remove ../../../